src/scaladoc/scala/tools/nsc/doc/html/HtmlPage.scala

```diff
@@ -145,8 +145,12 @@ abstract class HtmlPage extends Page { thisPage =>
         <span class="extmbr" name={ mbr.qualifiedName }>{ inlineToHtml(text) }</span>
     case Tooltip(tooltip) =>
       <span class="extype" name={ tooltip }>{ inlineToHtml(text) }</span>
-    case LinkToExternal(name, url) =>
-      <a href={ url } class="extype" target="_top">{ inlineToHtml(text) }</a>
+    case LinkToExternalTpl(name, baseUrl, dtpl: TemplateEntity) =>
+      def url = Page.makeUrl(baseUrl, Page.templateToPath(dtpl)) + name
```

Comment:
Internal links seem to be handled the same way (append `#` then the signature without any encoding). I'd also be wary of encoding it as a URL as the forbidden characters are not the same. What would you encode it with?